macos/docs/Controls/Link.md

```diff
@@ -0,0 +1,54 @@
+# Link
+
+## Overview
+With a Link, users can navigate to another page, window, or Help topic; display a definition; initiate a command; or choose an option. A Link indicates that it can be clicked, typically by being displayed using the visited or unvisited link system colors. Traditionally, Links are underlined as well, but that approach is often unnecessary and falling out of favor to reduce visual clutter.
+
+A Link is the lightest weight clickable control, and is often used to reduce the visual complexity of a design.
+
+
+
+## Best Practices
+### Do
+- Use to navigate to another page, window, or help topic; display a definition; initiate a command; or choose an option.
+- Make Links discoverable by visual inspection alone. Users shouldn't have to interact with your program to find links.
+- Use Links that give specific descriptive information about the result of clicking on the link, using as much text as necessary. Users should be able to - accurately predict the result of a link from its link text and optional Tooltip.
+- Use text that suggests clicking, such as a command starting with an imperative verb like "Show", "Print", "Copy", or "Delete".
+
+### Don't
+- Use if the action is destructive or irreversible. Because users associate links with navigation (and the ability to back out), Links aren't appropriate for commands with significant consequences.
```

Comment:
All of this is lifted from the [FluentUI Web Link](https://developer.microsoft.com/fluentui#/controls/web/link) and [FluentUI React Native Link](https://developer.microsoft.com/fluentui#/controls/crossplatform/link) documentation (which happens to be the same). All that was removed was a reference to a tooltip.

Reply:
The overview doesn't feel like it applies very well here. We don't support "visited" or "unvisited" link colors, it's really only used for "opening a given URL in an external browser", not "navigating to another page, window, or Help topic..." etc.